bicep/install.sh

```diff
@@ -201,7 +201,7 @@ if [ "$USE_INSIDERS_BUILD" == "true" ] || [ "$MANUAL" == "true" ]; then
     fi
 fi
 
-python3 /opt/ccw/cyclecloud_install.py --acceptTerms \
+python3 /opt/ccw/cyclecloud_install.py --acceptTerms --acceptMarketplaceTerms \
```

Comment:
We need the existence of this argument to be tied a value from the ccwOutputs.json. The default behavior has to be that we do NOT accept mp terms. The user must select an option in the UI or set an explicit setting in the bicep params. The default in the UI and in bicep params should be false.